### PR TITLE
fix: add upgrade step for missing stopwords registry entries

### LINKED ISSUE #371: Missing upgrade step since 9.2.0 for stopwords registry entries
When updating to 9.2.0 from older version, you get this error when going to @@solr-controlpanel

> KeyError: 'Interface `collective.solr.interfaces.ISolrSchema` defines a field `stopwords`, for which there is no record.'

There is an upgrade step missing since https://github.com/collective/collective.solr/pull/368 for those 2 registry entries ( stopwords and stopwords_case_insensitive ) .

### PR DESCRIPTION
**Pull request by remdub**
Closes #371

### CODE REVIEW

File: src/collective/solr/upgrades.zcml [hunk 1]
```diff
@@ -63,4 +63,14 @@
     profile="collective.solr:default"
     />
 
+  <genericsetup:upgradeStep
+    title="Upgrade to collective.solr 8"
+    description="Add stopwords registry entries"
+    source="7"
+    destination="8"
```
Comment (tisto) [inline src/collective/solr/upgrades.zcml:70]: @remdub thanks for your contribution! The current number in metadata.xml is 7:

https://github.com/collective/collective.solr/blob/main/src/collective/solr/profiles/default/metadata.xml#L3

Don't we need to upgrade that number as well then?

**Review by reebalazs — APPROVED** [review]
Hmm ok thank you, I've missed this.

**Comment by reebalazs** [comment]
@rendub can you please check the metadata issue that @tisto mentioned?

**Review by reebalazs — CHANGES_REQUESTED** [review]
pls check the metadata number in metadata.xml and test out if the upgrade is correct

**Comment by remdub** [comment]
> pls check the metadata number in metadata.xml and test out if the upgrade is correct

Thank you. Done and tested !

> 2024-02-07 09:16:17,736 INFO    [collective.solr:144][waitress-3] Migrated to version 8
2024-02-07 09:16:17,736 INFO    [GenericSetup:1138][waitress-3] Ran upgrade step Upgrade to collective.solr 8 for profile collective.solr:default

**Review by tisto — APPROVED** [review]

**Comment by tisto** [comment]
@remdub released with c.solr 9.2.3.

**Pull request merged**